app/assets/stylesheets/helpers/_contents-list.scss

```diff
@@ -0,0 +1,91 @@
+.contents-list {
+  margin-bottom: $gutter-two-thirds;
+
+  // Always render the contents list above a
+  // back to contents link
+  position: relative;
+  z-index: 1;
+  background: $white;
+  box-shadow: 0 20px 15px -10px $white;
+
+  li {
+    padding-top: $gutter / 4;
+    line-height: 1.3;
```

Comment:
https://css-tricks.com/almanac/properties/l/line-height/#article-header-id-0

> Unitless line heights are recommended due to the fact that child elements will inherit the raw number value, rather than the computed value. With this, child elements can compute their line heights based on their computed font size, rather than inheriting an arbitrary value from a parent that is more likely to need overriding.